fuel/schemes.py

```diff
@@ -18,6 +18,12 @@ class IterationScheme(object):
     sequential batches, shuffled batches, etc. for datasets that choose to
     support them.
 
+    Attributes
+    ----------
+    requests_examples : bool
```

Comment:
Simply `example_requests`? To be honest, I find the current name perfectly clear, in the sense that the iteration scheme is in a sense *requesting* examples or batches from the stream.